enterprise/config/BUILD

```diff
@@ -10,7 +10,7 @@ filegroup(
             "buildbuddy.release.yaml",
             "executor.release.yaml",
         ],
-        "//conditions:default": glob(["*.yaml"]),
+        "//conditions:default": glob(["**/*.yaml"]),
```

Comment:
If you run `git clean` then it'll delete your user configs - wdyt about putting configs in a parent dir like `--config_file=../my_config.yaml` ?